subsys/net/lib/wifi_credentials/wifi_credentials.c

```diff
@@ -137,7 +137,9 @@ int wifi_credentials_get_by_ssid_personal_struct(const char *ssid, size_t ssid_l
 	    buf->header.type != WIFI_SECURITY_TYPE_PSK_SHA256 &&
 	    buf->header.type != WIFI_SECURITY_TYPE_SAE &&
```

Comment:
I can also see other SAE types missing, can you please review the enumerations with Wi-Fi shell?

Reply:
Done.